.gitignore

```diff
@@ -1,4 +0,0 @@
-_site/
-.sass-cache/
-.jekyll-metadata
-*~
```

Comment:
Yes. I since have a radioactive-decay branch and an atomic-structure branch. Each only has its respective node now. If I understood correctly, then it's fixed now.